_typos.toml

```diff
@@ -0,0 +1,8 @@
+[default]
+extend-ignore-re = [
```

Comment:
Would you mind dropping a comment here about what these REs are supposed to ignore? Anyone who comes next to add or debug something will love you (including future you, I think 😄). In TOML, you can do

```toml
extend-ignore-re = [
    '`[^`\n]+`',  # skip ...?
    '```[\s\S]*?```',  # skip code blocks
]
```

Reply:
i believe the first one is an inline code block

Reply:
Aah, yes, that would be it 💡